src/modules/fundingManager/bondingCurveFundingManager/BondingCurveFundingManagerBase.sol

```diff
@@ -20,58 +20,82 @@ import {ERC20Upgradeable} from "@oz-up/token/ERC20/ERC20Upgradeable.sol";
 
 // External Libraries
 import {SafeERC20} from "@oz/token/ERC20/utils/SafeERC20.sol";
-import {Strings} from "@oz/utils/Strings.sol";
-
-import {IFundingManager} from "src/modules/fundingManager/IFundingManager.sol";
 
+/// @title Bonding Curve Funding Manager Base Contract.
+/// @author Inverter Network.
+/// @notice This contract enables the base functionalities for issuing tokens along a bonding curve.
+/// @dev The contract implements functionalties for:
+///         - opening and closing the issuance of tokens.
+///         - setting and subtracting of fees, expressed in BPS and subtracted from the collateral.
+///         - calculating the issuance amount by means of an abstract function to be implemented in
+///             the downstream contract.
 abstract contract BondingCurveFundingManagerBase is
     IBondingCurveFundingManagerBase,
-    IFundingManager,
```

Comment:
This ties to the comment above. Probably a good call, if IBondingCurveFundingManagerBase becomes a IFundingManager, it can call self.token() and avoid getting it through the proposal, which will be deprecated anyway. What do you think @Zitzak  ?

Reply:
I think it makes sense indeed.

The redeeming contract however also has FundingManagerBase in the title. Would you suggest renaming it? @FHieser

Reply:
I think since it inherits from bondingCurvefundingManagerBase (and therefore inherits IFundingManager) it's ok to keep the name as is